xml/System.Security.Cryptography/SHA256.xml

```diff
@@ -37,9 +37,7 @@
   
  The hash size for the <xref:System.Security.Cryptography.SHA256> algorithm is 256 bits.  
   
- This is an abstract class. The only implementation of this class is <xref:System.Security.Cryptography.SHA256Managed>.  
-  
-   
+ This is an abstract class. The .NET Framework provides 3 implementations derived from this class. 
```

Comment:
Doesn't the class hierarchy table show that?  I agree with removing the bad sentence, and I hope we never add another public type deriving from this, but why write down a number here that has to be kept correct?

Reply:
I agree with your remark. As mentioning the implementations seems useless, due to the class hierarchy shown above these informations, I remove the second sentence.

Reply:
This documentation is not just for .Net Framework, it's also for .Net Core, .Net Standard and Xamarin. So, I would prefer it if any explicit mentions of .Net Framework were avoided.